src/libs/kata-types/src/config/hypervisor/mod.rs

```diff
@@ -447,6 +447,12 @@ pub struct DeviceInfo {
     #[serde(default)]
     pub default_bridges: u32,
 
+    /// Enable PCI hotplug support, default false
+    ///
+    /// Enabling this will result in hotplugin VFIO devices (see below).
+    #[serde(default)]
+    pub pci_hotplug: bool,
+
```

Comment:
could you add a comment here that this currently is a dragonball-only config, so that other people could know that this config won't take effect on other VMMs?

Reply:
ack, thanks!